platforms/nuttx/src/bootloader/microchip/mpfs/main.c

```diff
@@ -878,6 +877,7 @@ static int loader_main(int argc, char *argv[])
 #endif
 
 #if defined(CONFIG_OPENSBI) && defined(CONFIG_MMCSD)
+	ssize_t uboot_size = 0;
```

Comment:
Unrelated change but who cares

Reply:
Yes.. I did this because I temporarily needed some more space for the bootloader, so I wanted to remove OPENSBI (px4-only boot). This caused build error because of this (unused variable)

I left it in the PR so that it doesn't get forgotten.. bug anyway